packages/pwa-kit-react-sdk/src/ssr/universal/hooks/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* istanbul ignore file */
+
+import React, {useContext} from 'react'
+import {CorrelationIdContext, ExpressContext} from '../contexts'
+
+export const useExpress = () => {
+    return useContext(ExpressContext)
+}
```

Comment:
🚫

Reply:
Good catch! I've removed the context and its context